tests/Integration/parsers/LocationParserTest.php

```diff
@@ -64,8 +64,7 @@ public function testGivenTitleThatIsLink_linkIsSetAndTitleIsNot( $link ) {
 
 	public function linkProvider() {
 		return [
-			[ 'https://semantic-mediawiki.org' ],
-			[ 'http://www.semantic-mediawiki.org' ],
```

Comment:
Would have been better to keep this. It is test code. Could just have well had test.com instead of smw.org.

Reply:
I was indeed a bit wondering about this. Well two instead of three test links should do, too.